test-resources/kt/feature_flag_system_2/control/configurations/rules.toml

```diff
@@ -128,7 +128,7 @@ query = """
                                                                        (simple_identifier) @arg_method_receiver
                                                                     (navigation_suffix (_)@arg_method_name)
                                                                       )
-                                                              )))))@ce
```

Comment:
The changes logic (below), now disallows wrong rules that reference non-existing variables